cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -1024,6 +1024,8 @@ export function getJobAsync(tid: number, jid: number, initialFrame: number, init
 
             loadJobEvent.close(await jobInfoGenerator(job));
 
+            state.annotation.canvas.instance.destroy();
```

Comment:
I believe, better to call this method in reducer before we reinitialize canvas. 
Looking this code, not clear why we should destroy current canvas.

Reply:
Moved it back to reducer